vendors/aws/simulate_cpu_load.js

```diff
@@ -0,0 +1,29 @@
+function simulateLoad(numberOfIterations) {
+  sum = 0;
+  for(var i=0; i<numberOfIterations; i++) {
```

Comment:
I'm wondering if adding a single number in a loop is not too simple to test out the load. With jitted JS it will end up with 2 instructions (`add` and `jmp` with comparison).

Reply:
Hmm, I believe that benchmarking these two instructions might reflect the performance of the hardware laying beneath. Well, I surely can add some calculation that would use some floating point arithmetic, but I'am not sure we might benefit from it.

Reply:
Let's leave it for now